### PR TITLE
More ILY responses

### PR DESCRIPTION
**Pull request by scpketer**

### CODE REVIEW

File: Monika After Story/game/script-topics.rpy [hunk 1]
```diff
@@ -3709,7 +3709,18 @@ label monika_love:
                 _("More than I can ever express!"),
                 _("It makes me so happy that you feel the same way I do!"),
                 _("I don't know what I would do without you!"),
-                _("You mean everything to me!")
+                _("You mean everything to me!"),
+                _("We're never gonna be apart!"),
```
Comment (jmwall24) [inline Monika After Story/game/script-topics.rpy:3713]:
```suggestion
```

File: Monika After Story/game/script-topics.rpy [hunk 2]
```diff
@@ -3709,7 +3709,18 @@ label monika_love:
...
+                _("We have so much to experience together!"),
+                _("I can't imagine my life without you!"),
+                _("I'm so happy to have you by my side!"),
+                _("We're really lucky to have each other!"),
+                _("You're my everything!"),
+                _("You give meaning to my entire life!"),
```
Comment (jmwall24) [inline Monika After Story/game/script-topics.rpy:3719]:
```suggestion
                _("You give meaning to my life!"),
```

**Review by jmwall24 — CHANGES_REQUESTED** [review]

File: Monika After Story/game/script-topics.rpy [hunk 3]
```diff
@@ -3709,7 +3709,18 @@ label monika_love:
...
+                _("I'm the happiest girl in the world!"),
+                _("I'll always be there for you."),
```
Comment (multimokia) [inline Monika After Story/game/script-topics.rpy:3721]:
```suggestion
                _("I'll always be here for you."),
```

File: Monika After Story/game/script-topics.rpy [hunk 4]
```diff
@@ -3709,7 +3709,18 @@ label monika_love:
...
+                _("I can't wait to feel your warmth!"),
+                _("Words cannot describe how I feel about you!")
```
Comment (multimokia) [inline Monika After Story/game/script-topics.rpy:3723]:
```suggestion
                _("Words can't describe how I feel about you!")
```

**Review by multimokia — CHANGES_REQUESTED** [review]

File: Monika After Story/game/script-topics.rpy (hunk 2 above)
Comment (multimokia) [reply Monika After Story/game/script-topics.rpy:3719]: Tbh, would kinda like this as `You're the meaning in my life!`

**Review by jmwall24 — APPROVED** [review]

**Review by multimokia — APPROVED** [review]

**Pull request merged**